test/video.test.js

```diff
@@ -11,8 +11,9 @@ describe('Video', () => {
 	beforeEach(() => {
 		containerEl = document.createElement('div');
 		containerEl.setAttribute('data-o-component', 'o-video');
-		containerEl.setAttribute('data-o-video-id', '4084879507001');
```

Comment:
Am I correct in thinking that if the `data-o-video-id` stayed as `4084879507001`, the test would still pass?